src/components/Auth/InputField/Input/Input.tsx

```diff
@@ -0,0 +1,33 @@
+import { ChangeEvent, FocusEventHandler } from "react";
+import styles from "./Input.module.scss";
+
+interface Input {
+  className: string;
+  type: string;
```

Comment:
이렇게 타입을 잡아주면 Input 컴포넌트를 사용할때 type 값을 입력하려고 하면 input의 type에서만 사용할 수 있는 속성값들이 보여질 것 같아요~
```suggestion
  type: HTMLInputTypeAttribute;
```

<img width="514" alt="image" src="https://github.com/codeit-bootcamp-frontend/4-Weekly-Mission/assets/51105841/50b65f4c-1f00-4200-a608-d25cebb345c2">